docs/cel2/notices/eigenda-v2-upgrade.md

```diff
@@ -0,0 +1,29 @@
+# EigenDA v2 Upgrade
+
+This page outlines changes related to the EigenDA v2 upgrade for node operators.
+
+:::info
+
+This page will be kept updated with key information about the upgrade. As this upgrade is activated on the sequencer, no detailed activation times can be given.
+
+- Baklava testnet activation is planned for Wed, Jul 30, 2025.
+- Alfajores testnet activation is tentative planned for Wed, Aug 20, 2025.
+- **Mainnet** activation is tentative planned for Wed, Sep 3, 2025.
+
+:::
+
+## What is the EigenDA v2 upgrade about?
```

Comment:
I would add:
- link to Eigen's doc about it
- link to the forum post where we announced it (Eclair?)